critiquebrainz/ws/review/views.py

```diff
@@ -283,10 +283,10 @@ def review_list_handler():
                 "id": "CC BY-NC-SA 3.0",
                 "info_url": "https:\/\/creativecommons.org\/licenses\/by-nc-sa\/3.0\/"
               },
-              "rating": 0,
               "source": "BBC",
               "source_url": "http:\/\/www.bbc.co.uk\/music\/reviews\/vh54",
               "text": "REVIEW TEXT GOES HERE",
+              "rating": 100, (Rating part of the review)
```

Comment:
This is what I asked about earlier. This should be done when making changes in the ws code for rating feature.